### PR TITLE
Feature/ui components

### PR DESCRIPTION
**Pull request by JessBrownless**

### CODE REVIEW

File: src/pages/blogs.js [hunk 1]
```diff
@@ -20,11 +20,11 @@ const Blogs = props => {
     <Layout>
       <SEO title="Blogs" />
       <div className="bg-gray-200">
-        <div className="container mx-auto pb-10 pl-6 pr-6 md:pl-0 md:pr-0">
-          <div className="grid grid-cols-1 pt-12 md:grid-cols-5 xl:grid-cols-7 md:gap-12">
+        <div className="container mx-auto pb-10 pl-6 pr-6">
+          <div className="grid grid-cols-1 pr-4 pl-4 pt-12 md:grid-cols-5 xl:grid-cols-7 md:gap-12">
```
Comment (sonnybooth-hexsu) [inline src/pages/blogs.js]: sort order of mqs. md is last here

File: src/pages/contact.js [hunk 2]
```diff
@@ -59,12 +61,12 @@ const Contact = () => {
                 <input type="hidden" name="form-name" value="contact" />
                 <input type="hidden" name="bot-field" />
                 <div className="flex flex-wrap -mx-3 mb-6">
-                  <div className="w-full md:w-1/2 px-3 mb-6 md:mb-0 lg:w-full">
+                  <div className="w-full md:w-1/2 px-3 mb-6 lg:w-full">
```
Comment (sonnybooth-hexsu) [inline src/pages/contact.js:64]: mq order needs sorting here

File: src/pages/index.js [hunk 3]
```diff
@@ -30,33 +31,32 @@ const IndexPage = () => (
         style={{ backgroundImage: `url(${heroImageMd})` }}
       >
         <div className="container mx-auto pl-6 pr-6">
-          <div className="pl-4 pr-4">
+        <div className="pl-4 pr-4">
             <div className="lg:w-1/2 flex flex-col items-center lg:items-start">
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: mq order

File: src/pages/index.js [hunk 4]
```diff
@@ -30,33 +31,32 @@ const IndexPage = () => (
...
-              </p> */}
+              <p className="text-sm text-center font-medium mb-8 md:w-2/3 lg:w-full lg:text-left lg:mb-8">
+                If you&lsquo;re looking for the latest in wireless headphones,
+                look no further. These are perfect for TV, stereo, home, and
+                cell phone.
+              </p>
               <div>
                 <a
-                  className={`${buttons.btnPrimary} block md:inline-block mb-4 md:mb-0 lg:mr-4`}
+                  className={`${buttons.btnPrimary} block lg:inline-block mb-6 md:mb-0 lg:mr-4 py-2 px-2`}
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: mq order

File: src/pages/index.js [hunk 5]
```diff
@@ -30,33 +31,32 @@ const IndexPage = () => (
...
                   href="#"
                 >
                   Book a consultation
                 </a>
                 <a
-                  className={`${buttons.btnTertiary} block md:inline-block`}
+                  className={`${buttons.btnTertiary} block lg:inline-block mb-6`}
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: mq order

File: src/pages/products.js [hunk 6]
```diff
@@ -21,8 +21,8 @@ const Products = props => {
     <Layout>
       <SEO title="Products" />
       <div className="bg-gray-200">
-        <div className="container mx-auto pl-6 pr-6 pb-10 md:pl-0 md:pr-0">
-          <div className="grid grid-cols-1 pt-12 md:grid-cols-5 xl:grid-cols-7 md:gap-12">
+        <div className="container mx-auto pl-6 pr-6 pb-10">
+          <div className="grid grid-cols-1 pr-4 pl-4 pt-12 md:grid-cols-5 xl:grid-cols-7 md:gap-12">
```
Comment (sonnybooth-hexsu) [inline src/pages/products.js:25]: mq order

File: src/assets/images/consultation.svg [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+            <circle id="Oval" cx="21" cy="3" r="1"></circle>
+        </g>
+    </g>
+</svg>
```
Comment (sonnybooth-hexsu) [inline src/assets/images/consultation.svg:17]: space at bottom of file

**Review by sonnybooth-hexsu — CHANGES_REQUESTED** [review]
some little changes

File: src/pages/blogs.js (hunk 1 above)
Comment (JessBrownless) [reply src/pages/blogs.js]: You did that

File: src/assets/images/consultation.svg (hunk 7 above)
Comment (JessBrownless) [reply src/assets/images/consultation.svg:17]: Can't see a space

**Pull request merged**